kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/rules/temp/drugbank/drugbank_rules.clj

```diff
@@ -1,99 +1,100 @@
-         
-         (?/inheres rdfs/subClassOf obo/CHEBI_23888) ;drug role
-         (?/inheres rdfs/subClassOf ?/r4)
+         (?/r_inheres rdf/type owl/Restriction)
+         (?/r_inheres owl/onProperty obo/RO_0000052) ;inheres_in
+         (?/r_inheres owl/someValuesFrom ?/drug)
+
+         (?/inheres rdfs/subClassOf obo/CHEBI_23888) ;drug (role)
+         (?/inheres rdfs/subClassOf ?/r_inheres)
 
          ;;make the interaction be necessarily part of these 3 restrictions
          ;;still need to add a sufficient definition
-         (?/interaction rdfs/subClassOf ?/r1)
-         (?/interaction rdfs/subClassOf ?/r2)
-         (?/interaction rdfs/subClassOf ?/r3))
-  
+         (?/interaction rdfs/subClassOf  ?/r_target_participant)
+         (?/interaction rdfs/subClassOf ?/r_drug_participant)
+         (?/interaction rdfs/subClassOf ?/r_realizes_drug_role))
 
-;;iaodrugbank/DrugBankDrugRecord_drugBankIdDataField1
-;;iaodrugbank/DrugBankPartnerRecord_externalIdentifiersDataField1
-  ;;(?/partnerRec obo/BFO_0000051 ?/entityFV)
-  
   :body
   (
-   ;; (_/fv0 kiao/hasTemplate iaodrugbank/DrugBankDrugRecord_drugBankIdDataField1)
-   ;; (_/fv0 obo/IAO_0000219 _/drugid)
-   ;; (_/record obo/BFO_0000051 _/fv0)
-
-   ;; (_/record obo/BFO_0000051 _/subrecord1)
-   ;; (_/subrecord1 obo/BFO_0000051 _/fv1)
-   ;; (_/fv1 kiao/hasTemplate iaodrugbank/DrugBankPartnerRecord_externalIdentifiersDataField1)
-   ;; (_/fv1 obo/IAO_0000219 _/externalid)
-   
-   (_/fv1 kiao/hasTemplate iaodrugbank/DrugBankPartnerRecord_externalIdentifiersDataField1)
-   (_/fv1 obo/IAO_0000219 _/externalid)
-   (_/partnerRecord1 obo/BFO_0000051 _/fv1) ; has_part
-   (_/targetRecord1 obo/BFO_0000051 _/partnerRecord1) ; has_part
-   (_/record obo/BFO_0000051 _/targetRecord1) ; has_part
-
-   (_/record obo/BFO_0000051 _/fv0) ; has_part
-   (_/fv0 kiao/hasTemplate iaodrugbank/DrugBankDrugRecord_drugBankIdDataField1)
-   (_/fv0 obo/IAO_0000219 _/drugid) ; denotes
+    ;; each DrugBank record has an ID field. We use that
+    ;; ID to link to the drug bioentity
+    (_/id_field kiao/hasTemplate iaodrugbank/DrugBankDrugRecord_drugBankIdDataField1)
+    (_/id_field obo/IAO_0000219 _/drug_id)
+  	(_/drug_id obo/IAO_0000219 ?/drug_bioentity)
 
-   (_/drugid obo/IAO_0000219 ?/drug) ; denotes
+    ;; the majority of DrugBank records link to a Target record.
+  	(_/base_record obo/BFO_0000051 _/id_field)
+  	(_/base_record obo/BFO_0000051 _/target_record)
+  	(_/target_record kiao/hasTemplate iaodrugbank/TargetSchema1)
 
-   (_/externalid obo/IAO_0000219 ?/bio) ; denotes
-   (?/bio [rdfs/subClassOf clojure.core/*] ?/gorgporv)
-   (?/gorgporv rdf/type kbio/GeneSpecificGorGPorVClass))
+    ;; the vast majority of targets contain a polypeptide record which
+    ;; links to a protein bio-entity
+    ;; of 12,704 target records, all but 225 have an associated polypeptide record
+    ;; we should look into those 225 targets at some point as they are currently excluded
```

Comment:
Do we need a card for this? If one already exists, it might be useful to add the URL to the comment.